src/tcgwars/logic/impl/gen3/PowerKeepers.groovy

```diff
@@ -2006,31 +2067,30 @@ public enum PowerKeepers implements LogicCardInfo {
           energyCost C, C, C
           onAttack {
             damage 70
-            flip { cantUseAttack(thisMove, self) }
+            flip 1, {}, { cantUseAttack(thisMove, self) }
           }
         }
       };
       case METAGROSS_EX_95:
       return evolution (this, from:"Metang", hp:HP150, type:M, retreatCost:4) {
         weakness R
+        weakness F
         resistance G, MINUS30
         pokePower "Magnetic Redraw", {
           text "Once during your turn (before your attack), if Metagross ex is your Active Pokémon, you may use this power. Each player shuffles his or her hand into his or her deck. Then, each player draws 4 cards. This power can't be used if Metagross ex is affected by a Special Condition."
           actionA {
             checkLastTurn()
             checkNoSPC()
             assert self.active : "$self is not an Active Pokémon"
+            assert ( [my, opp].any{curPlayer -> curPlayer.hand || curPlayer.deck} ) : "No player has cards in their deck or in their hand (...how?)"
```

Comment:
That statement in parenthesis isn't necessary. It may be a rare case, but it could happen. Why didn't the player using the PokéPower just pass in that case? Who knows. Why do PTCGO players play solitaire when they can just attack.

Reply:
Eh, just a little joke comment 😛 

There's def 99% chances of this never happening. It'd require all 60 cards (or 54 if not counting initial prizes) to be either in some PCS in play, as a stadium placed, or in the lost zone.

Reply:
Or in the Discard Pile

Reply:
Not sure why I specifically ignored that being a possibility. Still, it's an unlikely situation and a minor note, but I'll remove it 🤷